blocks/blockstore/bloom_cache_test.go

```diff
@@ -104,11 +104,11 @@ func TestHasIsBloomCached(t *testing.T) {
 	block := blocks.NewBlock([]byte("newBlock"))
 
 	cachedbs.PutMany([]blocks.Block{block})
-	if cacheFails != 2 {
-		t.Fatalf("expected two datastore hits: %d", cacheFails)
+	if cacheFails != 1 {
```

Comment:
I verified this was correct, the change was due to an implementation detail.  Unfortually, I can't remember why.  If it is important enough I will spend an hour or two looking into it.